### ISSUE TITLE
Collaboration Cycle for [Team Name, Product Name, Feature Name]

### ISSUE THREAD

**Issue by baltes** [department-of-veterans-affairs/va.gov-team#74306]
### Verify information about your product or feature in the VFS Product Directory.

- [X] I have reviewed the [VFS Product Directory](https://depo-platform-documentation.scrollhelp.site/getting-started/vfs-product-directory) and confirm that my information is correct (or that I have taken appropriate actions to update or add my product information to the directory).

### VFS team name

IVC Forms

### Product name

10-7959c CHAMPVA Other Health Insurance (OHI) Certification

### Feature name

Initial form build

### GitHub label for team

ivc-forms

### GitHub label for product

10-7959c

### Public DSVA Slack channel for VFS team

#ivc-forms

### When did/will you start working on this product/feature?

December 2024

### Will your work result in visible changes to the user experience?

Yes

### Are you doing research with VA.gov users?

Yes

### Will your work involve changes to...

Tools, applications and dynamic pages

### Does your product/feature have Google Analytics tracking and a KPI dashboard in Domo?

No

### Do you need to capture any additional analytics or metrics?

Yes

### Product outline

https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/champva/10-7959c/product-outline-10-7959c.md

### Verify all relevant materials provided to Governance Team

- [X] I have provided all relevant and up-to-date links, screenshots, images, designs, etc. of the as-is version of the product

### Add the GitHub labels for your team, product, and feature to this ticket.

- [X] I acknowledge that I must add the GitHub labels for my team, product, and feature to this ticket.

### Notify the Collaboration Cycle team of this ticket in your team's public Slack channel.

- [X] I acknowledge that I must notify the Governance team by completing the Collab Cycle Kickoff workflow in my team's Slack channel after submitting this issue. This can be completed by typing "/collab cycle" and selecting the kickoff workflow.

**Comment by baltes** [comment]
Hi @shiragoodman, I was looking to start updating this ticket to plan for staging review in the new year, but it looks like it's missing a lot of the content for the touchpoints above. Should I try to add those back, or is there a better way to resolve it?

**Comment by shiragoodman** [comment]
@baltes yes, looks like something went wrong. We have a script for updating our open Collab Cycle Request tickets if/when content changes, and looks like something went awry. Let me update this and I'll let you know when it's ready.

**Comment by shiragoodman** [comment]
@baltes ok, I was able to restore the last modified version from May, so all of the Midpoint Review information is still there (although formatting is a little funky). Staging Review section has been added and includes all of our current artifact requests. 

Thank you for flagging this!

FYSA @it-harrison

**Comment by baltes** [comment]
Great, thanks @shiragoodman!

**Comment by jamiefiore** [comment]
@baltes Accessibility testing to Staging is complete. Here's[ the ticket](https://github.com/orgs/department-of-veterans-affairs/projects/1533/views/1?sliceBy%5Bvalue%5D=Sprint+7&pane=issue&itemId=91256809&issue=department-of-veterans-affairs%7Cva.gov-team%7C99302)
